sources/models/src/aws-k8s-1.23-nvidia/defaults.d/50-kubernetes-aws.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/kubernetes-aws.toml
```

Comment:
For the lack of a better place to add this, you are missing a configuration file introduced by #2076:

```
containerd-cri-pki.toml
```